app/components/MazemapEmbed/index.tsx

```diff
@@ -176,6 +184,7 @@ export const MazemapEmbed = ({ mazemapPoi, ...props }: Props) => {
           touchAction: 'pan-x pan-y',
         }}
         id="mazemap-embed"
+        className={styles.mazemapEmbed}
```

Comment:
I don't understand what this styling is affecting? The span below? Because that has already a color: #111 by the `blockingText` class?

Reply:
It makes the color of the other text in the embed also black, even on dark mode. Previously zLevelControl (floor selector) got white text on white bg on dark mode.